oak/server/rust/oak_runtime/src/node/wasm/tests.rs

```diff
@@ -25,17 +25,32 @@ use oak_abi::{
 };
 use wat::{parse_file, parse_str};
 
-fn start_node(wasm_module: Vec<u8>, entrypoint_name: &str) -> Result<(), OakStatus> {
+fn start_node(
+    wasm_module: Vec<u8>,
+    entrypoint_name: &str,
+    signature: Option<Signature>,
```

Comment:
Seems simpler and also more realistic to just pass `Vec<Signature>` (or `&[Signature]`).

Reply:
Done